src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,17 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  color: $main-accent-color;
+  font-size: $main-font-size;
+  line-height: $main-line-height;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+h2,
+p {
+  margin: 0;
+}
```

Comment:
I want to reset browser's settings for all these selectors, even in bigger projects, so I don't have to do this for every single class

Reply:
Yes, u can do it, also good point 👍